sdk/servicebus/azure-servicebus/azure/servicebus/management/_model_workaround.py

```diff
@@ -244,8 +244,7 @@
 ### End of code generated by the script.
 
 
-def avoid_timedelta_overflow(td):
-    # type: (Optional[Union[timedelta, str]]) -> Optional[Union[timedelta, str]]
+def avoid_timedelta_overflow(td: Optional[timedelta]) -> Optional[timedelta]:
```

Comment:
was the previous union with str wrong?

Reply:
only place this is called was in management/_models and it looks like everything passed in is timedelta